### PR TITLE
i18n(es): `publish-to-npm.md`

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

- Adds `publish-to-npm.md` in spanish
- Updates page name in nav.ts

### CODE REVIEW

File: src/pages/es/guides/publish-to-npm.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Publica en NPM
+description: Aprenda a publicar componentes de Astro en NPM
+i18nReady: true
+---
+
+¿Construyendo un nuevo componente Astro? **¡Publicalo en [npm](https://npmjs.com/)!**
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:8]:
```suggestion
¿Construyendo un nuevo componente Astro? **¡Publícalo en [npm](https://npmjs.com/)!**
```

File: src/pages/es/guides/publish-to-npm.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+Publicar un componente de Astro es una excelente manera de reutilizar tu trabajo en tus proyectos y compartirlo con la comunidad de Astro. Los componentes de Astro se pueden publicar e instalar directamente desde NPM, como cualquier otro paquete de JavaScript.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:10]:
```suggestion
Publicar un componente de Astro es una excelente manera de reutilizar tu trabajo y compartirlo con la comunidad de Astro. Los componentes de Astro se pueden publicar e instalar directamente desde NPM, como cualquier otro paquete de JavaScript.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+¿Buscas inspiración? Vea algunos de nuestros [temas](https://astro.build/themes/) y [componentes](https://astro.build/integrations/) favoritos de la comunidad de Astro. También puedes [buscar npm](https://www.npmjs.com/search?q=keywords:astro-component) para ver el catálogo completo.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:12]:
```suggestion
¿Busca inspiración? Vea algunos de nuestros [temas](https://astro.build/themes/) y [componentes](https://astro.build/integrations/) favoritos de la comunidad de Astro. También puede [buscar en npm](https://www.npmjs.com/search?q=keywords:astro-component) para ver el catálogo completo.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+:::tip[¿No quieres hacerlo solo?]
+Consulte nuestra [plantilla de componente de Astro](https://github.com/astro-community/component-template) para obtener una plantilla lista para usar y mantenida por nuestra comunidad.
+:::
+
+## Inicio rápido
+
+Para comenzar a desarrollar sus componentes rápidamente, tenemos una plantilla configurada para usted.
+
+```bash
+# Inicializar la plantilla de componentes de Astro en una nueva carpeta
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:23]:
```suggestion
# Inicializar la plantilla de componentes de Astro en una nueva carpeta
# npm
```

File: src/pages/es/guides/publish-to-npm.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+```json
+{
+  "name": "mi-proyecto",
+  "workspaces": ["demo", "packages/*"]
+}
+```
+
+En este ejemplo, se pueden desarrollar varios paquetes juntos desde el directorio `packages`. También se puede hacer referencia a estos paquetes desde `demo`, donde puedes instalar una copia funcional de Astro.
+
+```shell
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:68]:
```suggestion
```shell
# npm
```

File: src/pages/es/guides/publish-to-npm.md [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+
+```json
+{
+  "files": ["index.js", "MyAstroComponent.astro", "MyReactComponent.jsx"]
+}
+```
+
+#### `keywords`
+
+Una serie de palabras clave relevantes para tu componente que se utilizarán para ayudar a otros usuarios a [encontrar su componente en npm](https://www.npmjs.com/search?q=keywords:astro-component) y en cualquier otro catálogo de búsqueda.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:161]:
```suggestion
Una serie de palabras clave relevantes para su componente que se utilizarán para ayudar a otros usuarios a [encontrar su componente en npm](https://www.npmjs.com/search?q=keywords:astro-component) y en cualquier otro catálogo de búsqueda.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+
+Recomendamos agregar `astro-component` como una palabra clave especial para maximizar su descubrimiento en el ecosistema de Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:163]:
```suggestion
Recomendamos agregar `astro-component` como palabra clave especial para maximizar su descubrimiento en el ecosistema de Astro.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+---
+<MyAstroComponent />
+<MyReactComponent />
+```
+
+---
+
+## Desarrollando tu paquete
+
+Astro no tiene un "modo de paquete" dedicado para el desarrollo. En su lugar, debe usar un proyecto demo para desarrollar y probar tu paquete dentro de tu proyecto. Este puede ser un sitio web privado que solo se use para desarrollo, o un sitio web público de demostración/documentación para su paquete.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:225]:
```suggestion
Astro no tiene un "modo de paquete" dedicado para el desarrollo. En su lugar, debe usar un proyecto demo para desarrollar y probar su paquete dentro de su proyecto. Este puede ser un sitio web privado que solo se use para desarrollo, o un sitio web público de demostración/documentación para su paquete.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+## Probando tus componentes
+
+Astro actualmente no envía un test runner. Esto es algo que nos gustaría abordar. _(Si está interesado en ayudar, [¡únase a nosotros en Discord!](https://astro.build/chat/))_
+
+Mientras tanto, nuestra recomendación actual para las pruebas es:
+
+1. Agregue un directorio `fixtures` de prueba a su directorio `demo/src/pages`.
+2. Agregue una nueva página para cada prueba que desee ejecutar.
+3. Cada página debe incluir un uso distinto de los componente que le gustaría probar.
+4. Ejecute `astro build` para construir tus fixtures, luego compare los resultados en el directorio `dist/__fixtures__/` con lo que esperabas.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:238]:
```suggestion
4. Ejecute `astro build` para construir sus fixtures, luego compare los resultados en el directorio `dist/__fixtures__/` con los resultados esperados.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+  ├─ test-name-01.astro
+  ├─ test-name-02.astro
+  └─ test-name-03.astro
+```
+
+## Publicando tu componente
+
+Una vez que tengas tu paquete listo, ¡puedes publicarlo en npm!
+
+Para publicar un paquete en npm, utilice el comando `npmpublish`. Si eso falla, asegúrese de haber iniciado sesión a través de `npm login` y que su `package.json` sea correcto. Si tienes éxito, ¡ya está!
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:251]:
```suggestion
Para publicar un paquete en npm, utilice el comando `npm publish`. Si eso falla, asegúrese de haber iniciado sesión a través de `npm login` y que su `package.json` sea correcto. Si tienes éxito, ¡ya está!
```

File: src/pages/es/guides/publish-to-npm.md [hunk 11]
```diff
@@ -0,0 +1,289 @@
...
+
+Tenga en cuenta que no hubo un paso de `compilación` para los paquetes de Astro. Cualquier tipo de archivo compatible con Astro se puede publicar directamente sin un paso de compilación, porque sabemos que Astro es compatible con ellos de forma nativa. Esto incluye todos los archivos con extensiones como `.astro`, `.ts`, `.jsx` y `.css`.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:253]:
```suggestion
Tenga en cuenta que no hay un paso de `compilación` para los paquetes de Astro. Cualquier tipo de archivo compatible con Astro se puede publicar directamente sin un paso de compilación, porque sabemos que Astro es compatible con ellos de forma nativa. Esto incluye todos los archivos con extensiones como `.astro`, `.ts`, `.jsx` y `.css`.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+
+¡Comparte tu arduo trabajo agregando tu integración a nuestra [biblioteca de integraciones](https://astro.build/integrations/)!
+
+### Datos en `package.json`
+
+La biblioteca se actualiza automáticamente todas las noches, agregando todos los paquetes publicados en NPM con la palabra clave `astro-component`.
+
+La biblioteca de integraciones lee los datos `name`, `description`, `repository` y `homepage` de su `package.json`.
+
+¡Los avatares son una excelente manera de resaltar su marca en la biblioteca! Una vez que se publique su paquete, puedes [crear una issue en GitHub](https://github.com/withastro/astro.build/issues/new/choose) con tu avatar adjunto y lo agregaremos al listado.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:267]: "un" issue? "una" issue? Idk, I say "un" 🤣 

```suggestion
¡Los avatares son una excelente manera de resaltar su marca en la biblioteca! Una vez que se publique su paquete, puedes [crear un issue en GitHub](https://github.com/withastro/astro.build/issues/new/choose) con tu avatar adjunto y lo agregaremos al listado.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 13]
```diff
@@ -0,0 +1,289 @@
...
+
+:::tip
+¿Necesita anular la información que nuestra biblioteca lee de NPM? ¡No hay problema! [Archive un problema](https://github.com/withastro/astro.build/issues/new/choose) con la información actualizada y nos aseguraremos de que el "nombre", la "descripción" o la "página de inicio" personalizados se utiliza en su lugar.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:270]:
```suggestion
¿Necesita anular la información que nuestra biblioteca lee de NPM? ¡No hay problema! [Cree un issue](https://github.com/withastro/astro.build/issues/new/choose) con la información actualizada y nos aseguraremos de que el "nombre", la "descripción" o la "página de inicio" personalizados se utiliza en su lugar.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 14]
```diff
@@ -0,0 +1,289 @@
...
+:::
+
+### Colecciones
+
+Además de la palabra clave requerida `astro-component`, también se utilizan palabras clave especiales para organizar automáticamente los paquetes. Incluir cualquiera de las palabras clave a continuación agregará su integración a la colección en nuestra biblioteca de integraciones.
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:275]:
```suggestion
Además de la palabra clave requerida `astro-component`, también se utilizan otras palabras clave especiales para organizar automáticamente los paquetes. Incluir cualquiera de las palabras clave a continuación agregará su integración a la colección en nuestra biblioteca de integraciones.
```

File: src/pages/es/guides/publish-to-npm.md [hunk 15]
```diff
@@ -0,0 +1,289 @@
...
+| Análisis    | `analytics`                              |
+| CMS         | `cms`, `database`                        |
+| CSS + UI    | `css`, `ui`, `icon`, `icons`, `renderer` |
+| E-commerce  | `ecommerce`, `e-commerce`                |
+| Rendimiento | `performance`, `perf`                    |
+| SEO         | `seo`, `performance`, `perf`             |
+
+## Comparte
+
+Lo alentamos a compartir tu trabajo, y realmente nos encanta ver lo que crean nuestros talentosos Astronautas. ¡Ven y comparte lo que creas con nosotros en nuestro [Discord](https://astro.build/chat/) o menciona [@astrodotbuild](https://twitter.com/astrodotbuild) en un Tweet!
```
Comment (agustinmulet) [inline src/pages/es/guides/publish-to-npm.md:289]:
```suggestion
Lo alentamos a compartir su trabajo, y realmente nos encanta ver lo que crean nuestros talentosos Astronautas. ¡Ven y comparte lo que creas con nosotros en nuestro [Discord](https://astro.build/chat/) o menciona [@astrodotbuild](https://twitter.com/astrodotbuild) en un Tweet!
```

**Review by agustinmulet — COMMENTED** [review]
Minor suggestions 💪

**Comment by kevinzunigacuellar** [comment]
LGTM! 🚀

**Comment by Yan-Thomas** [comment]
Quadra merge today??? Congrats ES team! :clap:

**Pull request merged**